docs/source/release/v6.8.0/reflectometry.rst

```diff
@@ -7,10 +7,15 @@ Reflectometry Changes
 
 New Features
 ------------
-.. amalgamate:: Reflectometry/New_features
+- The ``MagnetismReflectometryReduction`` algorithm now has an option allowing the return of empty reflectivity curves.
```

Comment:
This algorithm should probably have some documentation as maintenance. Issue made #36161